underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -198,7 +198,8 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri,
       clientConf.setSignerOverride(conf.get(PropertyKey.UNDERFS_S3_SIGNER_ALGORITHM));
     }
 
-    AmazonS3Client amazonS3Client = new AmazonS3Client(credentials, clientConf);
+    AmazonS3Client amazonS3Client = (AmazonS3Client) AmazonS3Client.builder()
```

Comment:
Why it needs casting, can we use the concrete type in the whole `S3AUnderFileSystem`?

Reply:
`AmazonS3ClientBuilder.build()` returns type `AmazonS3` instead of `AmazonS3Client`. Casting seems to be the only way. I think it's an issue with how aws sdk is implemented.